zerver/lib/actions.py

```diff
@@ -906,6 +907,44 @@ def do_send_messages(messages):
     # intermingle sending zephyr messages with other messages.
     return already_sent_ids + [message['message'].id for message in messages]
 
+def add_reaction(user_profile, message, emoji):
```

Comment:
call this `do_add_reaction` -- that's our naming convention for things that execute an operation without doing any validation.